mojo-publish-plugin/build.gradle

```diff
@@ -14,7 +14,7 @@ plugins {
 }
 
 group 'com.flipkart.mojopublish'
-version '1.0.0'
+version '1.0.1-SNAPSHOT'
```

Comment:
Not too important: Generally path version is incremented only when we do minor bug fixes. 
Behaviour fixes should bump up minor version